style.css

```diff
@@ -39,51 +62,133 @@ header {
     min-height: 15rem;
 }
 
+.decorativeBox img {
+    min-width: 15rem;
+    min-height: 15rem;
+    max-height: 30rem;
+    max-width: 20rem;
+    border-radius: 200px;
+}
+
 .jumbotronTextbox {
     min-width: 15rem;
     min-height: 15rem;
     max-height: 40rem;
     max-width: 30rem;
+    margin-left: 4rem;
 }
 
-h1 {
+h2 {
     color: #ffffff;
     background-color: #000000;
 }
 
 .jumbotron p {
```

Comment:
make it so that the text has the same vertical space between each line and so that it all stays above the button element

Reply:
perhaps just check this again as i think this still needs fixing -- i tried removing the margin: left part and that made the text stay above the button